administrator/components/com_languages/src/Model/LanguageModel.php

```diff
@@ -100,7 +100,7 @@ protected function populateState()
      *
      * @param   integer  $langId  The id of the member to get.
      *
-     * @return  mixed  User data object on success, false on failure.
+     * @return  object|boolean  User data object on success, false on failure.
```

Comment:
```suggestion
     * @return  object|false  User data object on success, false on failure.
```

See #42972 and #43006